config.w32

```diff
@@ -224,6 +224,52 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (PHP_CLIENT_SIDE_ENCRYPTION != "no" && mongoc_ssl_found) {
+    mongoc_opts.MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION = 1;
+
+    PHP_MONGODB_CFLAGS += "\
+      /I" + configure_module_dirname + "/src/libmongocrypt/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt/kms-message/src \
+      /I" + configure_module_dirname + "/src/libmongocrypt-compat \
+    ";
```

Comment:
`PHP_MONGODB_CFLAGS` is already passed to `EXTENSION()` by this point, so I think we need to use `ADD_FLAG` to set these. Additionally, we'll want to define the KMS constants.

Reply:
An alternative to using `ADD_FLAG` is moving the `EXTENSION` call to the bottom of the file. I've seen this in another extension, but I haven't tested it. Calling `ADD_FLAG` seems to work at this time, so I believe this works fine for us.